zerver/models/custom_profile_fields.py

```diff
@@ -79,6 +79,9 @@ class CustomProfileField(models.Model):
     display_in_profile_summary = models.BooleanField(default=False)
     required = models.BooleanField(default=False)
 
+    # Whether a non-admin user can edit this field.
```

Comment:
This is imprecise -- "Whether regular users can edit this field on their own account" would be more clear. As it is, it could sound like you can edit it for any user if true.

Reply:
Modified the comment as suggested.